sys/Makefile

```diff
@@ -17,8 +17,8 @@ ifneq (,$(filter nomac,$(USEMODULE)))
     DIRS += net/link_layer/nomac
 endif
 ifneq (,$(filter transport_layer,$(USEMODULE)))
-	USEMODULE += udp
-	USEMODULE += tcp
+  USEMODULE += udp
+  USEMODULE += tcp
```

Comment:
This is unrelated for the RP, @OlegHahm but why this is done twice?
The mmodules are also included in `Makefile.dep` [1]

[1] https://github.com/RIOT-OS/RIOT/blob/master/Makefile.dep#L12